Cargo.toml

```diff
@@ -125,7 +125,7 @@ once_cell = "1.18"
 log = "0.4.17"
 mime = "0.3.16"
 percent-encoding = "2.3"
-tokio = { version = "1.0", default-features = false, features = ["net", "time"] }
+tokio = { version = "1.0", default-features = false, features = ["net", "time", "macros"] }
```

Comment:
This is only need for http3, right? We can just add `tokio/macros` to the `http3` features.